bibtex.y

```diff
@@ -82,8 +83,16 @@ func Parse(r io.Reader) (*BibTex, error) {
 	select {
 	case err := <-l.Errors: // Non-yacc errors
 		return nil, err
-	case err := <-l.ParseErrors:
-		return nil, err
+	case err := <-l.ParseErrors: //yacc errors
```

Comment:
This isn't technically true I think? The error returned here now can be yacc (syntax) error or variable not found